### PR TITLE
backport KnownNamespaceSchemaLocation

### PR DESCRIPTION
**Pull request by cebe**
backported from https://github.com/goetas-webservices/xsd-reader/pull/45 for version 0.2.x

### CODE REVIEW

**Comment by cebe** [comment]
@goetas I need a branch that point to latest 0.2.x tag, this should not go into master.

File: tests/ImportTest.php [hunk 1]
```diff
@@ -37,4 +37,12 @@ public function testBase()
 
         $this->assertSame($remoteAttr, $localAttrs[0]);
     }
-}
\ No newline at end of file
+
+    public function testKnownNamespaceLocationImport()
+    {
+//        $this->reader->addKnownNamespaceSchemaLocation('urn:example:profile-1.1', __DIR__.'/schema/profile-1.1.xsd');
```
Comment (goetas) [inline tests/ImportTest.php]: i think this should be un-commented

Comment (cebe) [reply tests/ImportTest.php]: yeah, I wanted to check travis to see that this fails. but it does not seem to run on this PR.

**Comment by goetas** [comment]
Added 0.2 branch and changed the base for the PR

**Comment by cebe** [comment]
Seems tests are failing. Will look into this in more detail on Monday.

**Comment by cebe** [comment]
Fixed it, this should be ready to merge now.

**Comment by cebe** [comment]
tests are passing:

https://travis-ci.org/goetas-webservices/xsd-reader/jobs/514562698

**Comment by goetas** [comment]
very good!

**Pull request merged**